app/containers/EditRequestPage.js

```diff
@@ -14,6 +14,7 @@ class EditRequestPage extends Component {
 
   onExecuteRequest(request) {
     this.props.executeRequest(request.method, request.url, request.headers.toJS(), request.body);
+    this.props.logRequest(request);
```

Comment:
I think it would be more idiomatic to add the request to the history somewhere in the reducers, because it's a state change in response to an action. That way it's also decoupled from the UI completely.

For example could be done [here](https://github.com/ruisalgado/pragma/blob/feature/add-request-history/app/actions/project.js#L122) in the `executeRequest` function, just after replacing the variables.